bootstrap/config/config.go

```diff
@@ -180,6 +181,13 @@ func (cp *Processor) Process(
 
 	// Now must load configuration from local file if any of these conditions are true
 	if !useProvider || !cp.providerHasConfig || cp.overwriteConfig {
+		commonConfigLocation := environment.GetCommonConfig(cp.lc, cp.flags.CommonConfig())
+		if commonConfigLocation != "" {
+			err := cp.loadCommonConfigFromUri(commonConfigLocation, serviceConfig, serviceType)
+			if err != nil {
+				return fmt.Errorf("could not load oommon config: %s", err.Error())
```

Comment:
error already has context
```suggestion
				return err
```